### ISSUE TITLE
DIA-NN exits and questions about "DIA-quant"

### ISSUE THREAD

**Issue by klannieha** [vdemichev/DiaNN#433]
Hi, 
I have generated a DDA spectral library following the DIA-SpecLib_Quant workflow. I am trying to quantify the DIA files in a separate run by providing the library.tsv file , however it shows an error that "No LC-MSMS files are found excluding diaPASEF" when i chose "DIA-quant" for the file type.
I have also try setting the file type as "DIA" and unchecked all the steps up until DIA-NN. The workflow exits with the log "Process 'DIA-NN' finished, exit code: -1073741515", and it doesn't seem like DIA-NN was executed at all. 

Version of Fragpipe is v18.0, and DIA-NN is version 1.8.1. Please let me know how to resolve these problem! Thank you!

**Comment by klannieha** [comment]
Wrong repo posted. Sorry.

**Comment by klannieha** [comment]
Sorry, wrong repository posted, will repost this on Fragpipe.

**Comment by vdemichev** [comment]
Yes, MSFragger-DIA currently does not support dia-PASEF, but DIA-NN does.
